docs/cudf/source/Working-with-missing-data.ipynb

```diff
@@ -0,0 +1,3032 @@
+{
```

Comment:
<p>This statement might be confusing since we added two dataframes above. May be little more explanation regarding the conditions in which this will be applicable might be better.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cudf/pull/7010/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
<p>Yep, introduced more clear text and example.</p>